supabase/pending/1_single_ops_collection.sql

```diff
@@ -0,0 +1,47 @@
+begin;
+
+-- Rename the existing ops tenant to the new specific ops tenant.
+update tenants set tenant = 'ops.us-central1.v1/' where tenant = 'ops/';
```

Comment:
do we need to update the ops role_grants , as well as the user_grant for support@estuary.dev ?

Reply:
Yep, good call. Also need to update the storage mappings for the ops tenant.

We need to keep the grants for `ops/` until the per-tenant deletion publications are complete. That along with the note above about re-creating the stats table in the same transaction as it is deleted led me to modify the migrations: There's still a second migration, but it is now all about updating the ops tenant - renaming the tenant itself, its grants, and its storage mappings.

There will be vestigial role grants for `$tenant` to read `ops/$tenant` left over in the system that we should clear out, as well as update the agent to stop creating sometime soon.